docs/src/extend/ways-to-extend.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Ways to Extend ESLint
+eleventyNavigation:
+    key: ways to extend
+    parent: extend eslint
+    title: Ways to Extend ESLint
+    order: 1
+---
+
+ESLint is highly pluggable and configurable. There are a variety of ways that you can extend ESLint's functionality.
+
+This page explains the ways to extend ESLint, and how these extensions all fit together.
+
+## Plugins
+
+Plugins let you add your own ESLint custom rules, processors, and formatters to a project. You can publish a plugin as a npm module.
+
+Plugins are useful because your project may require some ESLint configuration that isn't included in the core `eslint` package. For example, if you're using a frontend JavaScript library like React or framework like Vue, these tools have some features that require custom rules outside the scope of the ESLint core rules.
```

Comment:
```suggestion
Plugins are useful because your project may require some ESLint configuration that isn't included in the core `eslint` package. For example, if you're using a frontend JavaScript library like React or a framework like Vue, these tools have some features that require custom rules outside the scope of the ESLint core rules.
```